internal/host/common.go

```diff
@@ -29,6 +29,7 @@ const (
 	statusInfoResettingPendingUserAction                       = "Host requires booting into the discovery image to complete resetting the installation"
 	statusInfoPreparingForInstallation                         = "Host is preparing for installation"
 	statusInfoHostPreparationSuccessful                        = "Host finished successfully to prepare for installation"
+	statusInfoHostPreparationFailure                           = "Host failed to finished to prepare for installation"
```

Comment:
s/"Host failed to finished to prepare for installation"/"Host failed to prepare for installation"/